### PR TITLE
Flappy Learning (Workshop Bounty)

### PR DESCRIPTION
**Pull request by ImBIOS**
revisioning some of the sentences and improving to have more friendly words.
from #1569

### CODE REVIEW

File: workshops/flappy_learning/README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ var start = function(){
 
 Then we will create a neuron preceptor network. And we will set the population of the baby bird to learn to 50. You can tweak the network of course, but I recommending it as [2, [2], 1] or [4, [4], 1].
 
+_Neuron Preceptor is a network design of the brain of the Machine Learning, similar to human that have its brain cell that united to be a network that is called brain_
```
Comment (sampoder) [inline workshops/flappy_learning/README.md:51]: This is highly ambiguous and still doesn't actually tell me what this is

File: workshops/flappy_learning/README.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ Here is the [starter project](https://repl.it/@iMega/flappy-learning-starter#cod
 
 _^ You will see something like this._
 
-## NeuroEvolution Initialization
-Let's begin our coding! We start from our first variable declaration for the Neuroevolution, let's give it a name, like Neuvol and game variable for the game function. Then we initialize the start function.
+## First Line of Code
```
Comment (sampoder) [inline workshops/flappy_learning/README.md:30]: This title doesn't make sense to me.

**Review by sampoder — CHANGES_REQUESTED** [review]
View below

File: workshops/flappy_learning/README.md (hunk 2 above)
Comment (ImBIOS) [reply workshops/flappy_learning/README.md:30]: Any suggestions for me? What could be a cooler and sensible title?

File: workshops/flappy_learning/README.md (hunk 1 above)
Comment (ImBIOS) [reply workshops/flappy_learning/README.md:51]: Sorry for the bad sentences and definitions. I think it was cool. I will revisioning it.

Comment (ImBIOS) [reply workshops/flappy_learning/README.md:51]: please check it here #1599

File: workshops/flappy_learning/README.md (hunk 2 above)
Comment (ImBIOS) [reply workshops/flappy_learning/README.md:30]: please check it here #1599

**Comment by ImBIOS** [comment]
I have made some changes here #1599

**Pull request closed without merging**